### PR TITLE
feat: fix compilation on new Android arch

### PR DESCRIPTION
**Pull request by FlaviuExtPrimer**
https://primerapi.atlassian.net/browse/ACC-4531
https://primerapi.atlassian.net/browse/ACC-4511
https://primerapi.atlassian.net/browse/ACC-4629

### CODE REVIEW

**Comment by FlaviuExtPrimer** [comment]
Don't merge until branch-sdk-refactor is merged to primer-sdk-android/master.

TODO: Update primer-sdk-android version.

**Review by semirp — APPROVED** [review]

**Pull request merged**
